cypress/e2e/default/learn/header/universal-navigation.js

```diff
@@ -64,6 +73,79 @@ describe('Language menu', () => {
 });
 
+describe('Language Menu with keyboard', () => {
+  before(() => {
+    cy.get(selectors['exit-lang-menu']).click();
+    cy.get(selectors['navigation-list']).contains('Change Language').blur();
+  });
+
+  const langMenuOptionSelector =
+    selectors['language-menu'] + ' ' + selectors['lang-menu-option'];
+
+  it('should open Language Menu with ENTER key', () => {
+    cy.get(selectors['navigation-list'])
+      .contains('Change Language')
+      .type('{enter}');
+    cy.get(selectors['language-menu']).should('be.visible');
+  });
+  it('should open Language Menu with UP arrow key', () => {
+    cy.get(selectors['exit-lang-menu']).click();
+    cy.focused().type('{upArrow}');
+    cy.get(selectors['language-menu']).should('be.visible');
+    cy.get(langMenuOptionSelector).last().should('be.focused');
+  });
+  it('should open Language Menu with DOWN arrow key', () => {
+    cy.get(selectors['exit-lang-menu']).click();
+    cy.focused().type('{downArrow}');
+    cy.get(selectors['language-menu']).should('be.visible');
+    cy.get(langMenuOptionSelector).first().should('be.focused');
+  });
+  it('should navigate through language options with UP and DOWN keys', () => {
+    navigateLanguagesWithKey('downArrow');
+    navigateLanguagesWithKey('upArrow');
+  });
+  it('should move to last language option', () => {
+    cy.get(langMenuOptionSelector).first().focus();
+    cy.focused().type('{upArrow}');
+    cy.get(langMenuOptionSelector).last().should('be.focused');
+  });
+  it('should move to first language option', () => {
+    cy.get(langMenuOptionSelector).last().focus();
+    cy.focused().type('{downArrow}');
+    cy.get(langMenuOptionSelector).first().should('be.focused');
+  });
+  it('should close language options and focus on Change Language option', () => {
+    cy.focused().type('{esc}');
+    cy.get(selectors['language-menu']).should('not.be.visible');
+    cy.get(selectors['navigation-list'])
+      .contains('Change Language')
+      .should('be.focused');
+  });
+  it('should close language options with SHIFT+TAB', () => {
+    cy.focused().click();
+    // cy.tab() doesn't accept tabindex="-1" elements as focusable
+    // https://github.com/kuceb/cypress-plugin-tab/issues/18
+    // It is better to use .tab() on focused language option
+    // cy.focused().tab({ shift: true }); // uncomment this line when the issue is fixed
+    cy.get('body').tab({ shift: true }); // remove this line when the above issue is fixed
+
+    cy.get(selectors['language-menu']).should('not.be.visible');
+    cy.get(selectors['navigation-list'])
+      .contains('Sign in to change theme.')
+      .should('be.focused');
+  });
+
+  // cy.tab() doesn't accept tabindex="-1" elements as focusable
+  // https://github.com/kuceb/cypress-plugin-tab/issues/18
+  // Also, doing .tab() from element other than language option doesn't yield the expected result
+  it.skip('should close language options with TAB and focus on next tabbale element', () => {
```

Comment:
I think it's fine to skip this test for until the issue with the tab plugin is resolved. Thank you for writing a test in the meantime, though.

Just another suggestion for the test text:

```suggestion
  it.skip('when focused on an item in the language menu, pressing Tab should focus on the next tabbable element', () => {
```